src/examples/src/widgets/tree/Basic.tsx

```diff
@@ -0,0 +1,36 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Tree from '@dojo/widgets/tree';
+
+const factory = create();
+
+export default factory(function Basic() {
+	const nodes = [
```

Comment:
I think we need to flatten this and look at working with IDs.

Reply:
something like `{id: string, content: string, children: string[]}[]`  or track the parent ID

Reply:
I took the opposite approach: `parent?: string`. I think this better matches relational structures. Also used `value: string` as that is consistent with `ListOption`.